app/db.rb

```diff
@@ -126,6 +126,14 @@ class App < Sinatra::Base
     end
   end
 
+  put "/db/notes/:id" do
+    @record = Ronin::DB::Note.find_by(params[:id])
+
+    unless @record || @record.update(params)
```

Comment:
I would return a different error code if the `update` fails, like `400` (Bad Request).